.github/workflows/build.yml

```
@@ -115,6 +115,28 @@
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
+
+  nats:
+    runs-on: ubuntu-latest
+    services:
+      nats:
+        image: nats:latest
+        ports:
+          - 4222:4222
+    steps:
+    - uses: actions/checkout@v4
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)